split-llvm-extract.cpp

```diff
@@ -232,17 +302,19 @@ int main(int argc, char **argv) {
     outputFile.close();
   }
 
+  /**
+   * This is the forth and final stage.
```

Comment:
s/forth/fourth/ (you may, for amusement value, want to look up "painting the Forth bridge" :)).